packages/server/lib/modes/run.js

```diff
@@ -824,6 +824,10 @@ module.exports = {
       return
     }
 
+    if (quiet === true) {
+      return videoCapture.process(name, cname, videoCompression)
+    }
```

Comment:
this should be deduplicated with the invocation of videoCapture.process on line 904, either by a helper function or by refactoring postProcessRecording

Reply:
I've done this at cleanly as I know how, but I don't do a lot of JS work so let me know this is not 
idiomatic